### DISCUSSION TITLE
4.38 beta3 feedback

### DISCUSSION THREAD

**Discussion by Thanatosone** [authelia/authelia#6318] [Pre-Release Feedback]
Hi James:
Really like the reliability and lightweight for Authelia. I have been using it to protect several services without totp like AdguardHome. And always looking forward to have a web interface configurations panel. Although the yml file have already been cleared enough but it’s a little bit inconvenience to add new rules or modify settings when not in front of the server. Found that beta3 just upload the latest version to the docker hub for couple hours and already try it. So far haven’t found any bugs, and some field in the yml is a little bit different than version 4.37.5. For me the SMTP server have change the host. I change those line with old version 4.37.5 while seems like gmail not having the smtp server start with “smtp://“. It pop a warning message in log but it still works fine. The most excited thing I want to try is the web interface. The welcome page show without problems. Although so far haven’t seen any settings able to configure on the web interface, however the totp registration function work fine with email verification code. Hope to see more setting can be modify on the web interface. Again really appreciate your work and hope my feedback can make some help to improve Authelia.

**Comment by james-d-elliott** [comment]
Settings will only include user settings, administration actions will be a completely separate interface and probably so separate that you'd be able to run it on a separate hostname or server if you wanted.

You should include the warning message.

**Reply by Thanatosone** [reply]
Thanks for your quick reply. So what you means is there’s already have a separate administration setting page? If so then I must set it up with some mistake. I deployed it as usual like the 4.37.5 version on my Synology NAS. I was initially planned to try it on the localhost, I open 127.0.0.1:9091 and found that it can not be accessed without SSL. I then use the old domain name in NPM that I used to set for the 4.37.5 version. With the domain name and SSL I was able to see the login page. After log in with password set in userdata.yml, I am able to see the welcome page. Is there any mistake I’ve been operated? I didn’t found too much guide for the setting webpage on Authelia official website. As what you mentioned there should be another webpage can be accessed. I should set another domain name for it? Sorry didn’t try to dig it that much while only set the user setting webpage. I didn’t mention there’s warning messages since I was able to log in. Can you tell me how to access the administrator setting webpage? I’d be willing to try it again and hope to give some useful feedback when I have more time this weekend.

**Reply by james-d-elliott** [reply]
No we do not have an Admin UI, it's completely unrelated and has nothing to do with to the user settings is what I'm saying. We have plans for it but it's not as high a priority as other elements.

We do not support localhost deployments (best effort support only and unlikely to make changes to improve support here) and require HTTPS.

**Reply by Thanatosone** [reply]
Thanks for your explanation. Then I think I have already try the full settings UI. Although it's not that much category so far, however it still encouraging to see this huge step. We will keep be patient and looking foward to the admin UI. Again thanks for your amazing works and wish you got better quality of life (from the article Release Information know that you recently have higher leverage of work).
